pandas/tests/groupby/test_transform.py

```diff
@@ -834,3 +834,14 @@ def demean_rename(x):
     tm.assert_frame_equal(result, expected)
     result_single = df.groupby('group').value.transform(demean_rename)
     tm.assert_series_equal(result_single, expected['value'])
+
+
+@pytest.mark.parametrize('func', [min, max, np.min, np.max])
```

Comment:
Maybe parametrize first and last here as well?

Reply:
With a lambda function within `transform`?

Reply:
String would typically be the easiest application

Reply:
Ah I see. I forgot about those options. Thanks